emperor/support_files/js/sceneplotview3d.js

```diff
@@ -81,72 +86,83 @@ define([
      */
     this.visibleDimensions = _.clone(this.decViews.scatter.visibleDimensions);
 
-    /**
-     * Ranges for all the decompositions in this view (there's a min and a max
-     * property).
-     * @type {Object}
-     */
-    this.dimensionRanges = {'max': [], 'min': []};
-    this._unionRanges();
-
     // used to name the axis lines/labels in the scene
     this._axisPrefix = 'emperor-axis-line-';
     this._axisLabelPrefix = 'emperor-axis-label-';
 
-    // Set up the camera
-    var max = _.max(decViews.scatter.decomp.dimensionRanges.max);
-    var frontFrust = _.min([max * 0.001, 1]);
-    var backFrust = _.max([max * 100, 100]);
-
-    /**
-     * Camera used to display the scene.
-     * @type {THREE.PerspectiveCamera}
-     */
-    // these are placeholders that are later updated in updateCameraAspectRatio
-    this.camera = new THREE.OrthographicCamera(-50, 50, 50, -50);
```

Comment:
Would you mind re-adding this property? It looks like it is still in use but its underlying value will change depending on the view-type. Perhaps leave this as:

```javascript
this.camera = null;
```

And also make a note that this changes depending on what view is active. Seems to me that the `scatterCam` and the `paralleCam` could both be private properties?